front/app/containers/Admin/reporting/components/ReportBuilder/Analysis/Question.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import { Box, Icon } from '@citizenlab/cl2-component-library';
+
+import useAnalysisQuestion from 'api/analysis_questions/useAnalysisQuestion';
+
+import InsightBody from 'containers/Admin/projects/project/analysis/Insights/InsightBody';
+import InsightFooter from 'containers/Admin/projects/project/analysis/Insights/InsightFooter';
+import QuestionHeader from 'containers/Admin/projects/project/analysis/Insights/QuestionHeader';
+import { removeRefs } from 'containers/Admin/projects/project/analysis/Insights/util';
+
+import TextMultiloc from '../Widgets/TextMultiloc';
+
+import DraggableInsight from './DraggableInsights';
+
+const Question = ({
+  questionId,
+  analysisId,
+  projectId,
+  phaseId,
+  selectedLocale,
+}: {
+  questionId: string;
+  analysisId: string;
+  projectId: string;
+  phaseId?: string;
+  selectedLocale: string;
+}) => {
+  const { data } = useAnalysisQuestion({ analysisId, id: questionId });
+
+  const question = data?.data.attributes.question;
+  const answer = data?.data.attributes.answer;
+  const filters = data?.data.attributes.filters;
+  const generatedAt = data?.data.attributes.generated_at;
+
+  if (!question || !answer) {
+    return null;
+  }
+
+  return (
+    <Box
+      display="flex"
+      flexDirection="column"
+      justifyContent="space-between"
+      gap="16px"
+    >
+      <DraggableInsight
+        id="e2e-draggable-insight"
+        component={
+          <TextMultiloc text={{ [selectedLocale]: removeRefs(answer) }} />
+        }
+      >
+        <Icon name="menu" />
```

Comment:
It was not clear to me that the hamburger menu / three horizontal lines icon was the drag handle. Maybe we should add some copy explaining that that's where you start dragging? Also hovering over it, we could show the 'drag hand' mouse pointer, and maybe while it's dragging have a bit of text explaining what you're dragging (like the other widgets)?